tests/installation/bootloader_s390.pm

```diff
@@ -89,7 +89,9 @@ sub prepare_parmfile {
         $params .= " info=" . create_infofile("install: $instsrc");
     }
     else {
-        if (get_var('AGAMA')) {
+        if (get_var('AGAMA') && get_var('FLAVOR') == "Full") {
+            $params .= " root=live:ftp://" . get_var('REPO_HOST', 'openqa') . '/' . get_required_var('REPO_0') . "/LiveOS/squashfs.img";
+        } elsif (get_var('AGAMA')) {
```

Comment:
```suggestion
$params .= " root=live:ftp://" . get_var('REPO_HOST', 'openqa') . '/' . 
        (get_var('FLAVOR') eq "Full") ?
            get_required_var('REPO_0') . "/LiveOS/squashfs.img";
            get_var('REPO_999');
```
Ternary are much more clear and you don't have to repeat the string and neither you need to declare it.
And watch out for `==` use the string comparation operator, or exact regex or eq.